src/Orc.Controls.Examples/ViewModels/DatePickerViewModel.cs

```diff
@@ -21,6 +25,36 @@ public DatePickerViewModel()
 
         #region Properties
         public DateTime DateValue { get; set; }
+        public FastObservableCollection<object> AvailableFormats { get; private set; }
+        public object SelectedFormat { get; set; }
+        #endregion
+
+        #region Methods
+        protected override async Task InitializeAsync()
+        {
+            await base.InitializeAsync();
+
+            AvailableFormats = new FastObservableCollection<object>();
+            using (AvailableFormats.SuspendChangeNotifications())
+            {
+                foreach (var cultureInfo in CultureInfo.GetCultures(CultureTypes.AllCultures))
+                {
+                    var format = new
+                    {
+                        CultureCode = string.Format("[{0}]", cultureInfo.IetfLanguageTag),
+                        FormatValue = cultureInfo.DateTimeFormat.ShortDatePattern
+                    };
+
+                    AvailableFormats.Add(format);
+                    if (cultureInfo.Equals(CultureInfo.CurrentCulture))
+                    {
+                        SelectedFormat = format;
+                    }
+                }
+            }
+
+            await TaskHelper.Completed;
```

Comment:
This code won't do anything. You either have a task method without async (in that case, you must return a Task, which can be TaskHelper.Completed). But if you use await somewhere, this code can be removed.